src/webui/webutils.jl

```diff
@@ -13,43 +13,19 @@ function getcookie(r::HTTP.Request, key::AbstractString, default="")
     return ind === nothing ? default : cookies[ind].value
 end
 
+readtpl(tpl) = Mustache.template_from_file(joinpath(dirname(@__FILE__), "templates", tpl))
+const INDEX_TPL = readtpl("index.tpl")
+const SELECT_TPL = readtpl("select.tpl")
```

Comment:
I'm pretty sure making these things `const` means that they get compiled in and won't respond to updates in stuff like `route_prefix` etc.

Reply:
We pass routes as args to the templates while rendering so this is ok.